src/app/_services/configuration/configuration.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class ConfigurationService {
+    // TODO(CDuPlooy): For now localhost:8080 seems sane enough. Maybe change it to docks-api at some point.
+    private api_hostname = 'https://localhost:8080';
+    constructor() { }
+
+    public getAPIHostname(): string {
+        return this.api_hostname;
+    }
+
+    public setAPIHostname(hostname: string) {
+        this.api_hostname = hostname;
+        // Perhaps introduce an observable that fires everytime this value is changed
+        // for more complicated components. I don't really see a use case so I'm leaving this comment.
+
+        // Check if host is reachable before assigning the hostname ?
```

Comment:
It would be more flexible if the check is a separate function. Then it can also be used to check for further errors.